tests/roles/dataplane_adoption/tasks/neutron_verify.yaml

```diff
@@ -51,4 +51,4 @@
 - name: verify connectivity to the existing test VM instance using Floating IP
   when: prelaunch_test_instance|bool
   ansible.builtin.shell: |
-      ping -c4 {{ lookup('env', 'FIP') | default('192.168.122.20', True) }}
+      ping -c10 {{ lookup('env', 'FIP') | default('192.168.122.20', True) }}
```

Comment:
pasting from slack

```
can we maybe do a wait_for with an explicit timeout (like 3/4 mins or so?)
10:07
wait_for_connection
10:07
i was looking for ansible module that would help https://docs.ansible.com/ansible/latest/collections/ansible/builtin/wait_for_connection_module.html
10:08
so, as a first pass i think we can merge the proposal
10:08
and that can be a task for improvement
10:08
for future adoption ci team

```

Reply:
please remember that there is a rule for any shell command in tests to match the same commands in docs. This is why we should avoid ansible smartness in test suits